<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VhfVoice</class>
 <widget class="QWidget" name="VhfVoice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblVoice1_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblVoice2_1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblRadioName1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblRadioName2">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblVoice1_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>50</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lblVoice2_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>50</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="barSound">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>2</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>70</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblMute">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>40</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>音量</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
